README.md

```diff
@@ -1,19 +1,28 @@
-# Stanford Humanities And Sciences
+# Stanford School of Humanities And Sciences HSDP
 
-This is an Acquia BLT tool to assist in deploying code for Humanities and Sciences installation profile.
+This codebase runs the Humanities and Sciences Drupal (or Digital) Platform a.k.a. HSDP.  This platform runs 130+ sites for departments, programs, centers, councils, initiatives, etc. The platform is hosted on Acquia.
 
-## Getting Started
+## Overview
 
-This project is based on BLT, an open-source project template and tool that enables building, testing, and deploying Drupal installations following Acquia Professional Services best practices.
+This project is based on [Acquia BLT](https://docs.acquia.com/acquia-cms/add-ons/blt) (Bacon Lettuce Tomato, or Build and Launch Tool), an open-source project template and tool that enables building, testing, and deploying Drupal installations following Acquia Professional Services best practices.
 
-To set up your local environment and begin developing for this project, refer to the [BLT onboarding documentation](https://docs.acquia.com/blt/developer/onboarding/). Note the following properties of this project:
-* Primary development branch: develop
-* Local environment: DrupalVM
-* Local drush alias: @my-project.local
-* Local site URL: http://local.my-project.com
+Note the following properties of this project:
+* Primary development branch: `develop`
```

Comment:
This is not accurate. `develop` contains the latest _stable_ code. Feature branches are not merged into `develop`, instead, SWS creates a release branch for each sprint, using the `[VERSION]-release` naming pattern. For example, the current release branch is `11.2.3-release`. All the feature PR's should be created against the current release branch.

Also, not sure if this should be included in the documentation as is specific to Four Kitchens, but there are some other considerations for our PR's:
- Most of Four Kitchens PR's are grouped into a secondary release branch (`fk-stnfd-sprint-[SPRINT_NUMBER]`) that's delivered to SWS for review the Friday before the sprint ends. The purpose of this is to make the review process easier for SWS.
- In some cases, specially for larger PR's or long-lived branches, we skip the secondary release branch and send the PR directly to SWS, to prevent holding up the release branch in case there are kickbacks or further discussion is needed.

Reply:
I tweaked that section. Take another look.  I'm thinking that for the task at hand, we should focus on concise must-have info that won't change over time.  I think some of what you described in your comment isn't necessarily about "getting set up" so if we want to document that more (maybe we do, maybe we don't), it should go elsewhere in a future ticket.

Reply:
@dalin- Great, looks good. And I agree, let's keep things concise. I'll assign the PR to @joegl for the SWS review.